### PR TITLE
Calendar fixed and examples added (3-4 screen web & mobile, jsonforms)

### PR DESCRIPTION
**Pull request by sagar0211**

### CODE REVIEW

File: src/components/json-form/JS.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable react/prop-types */
```
Comment (saurshaz) [inline src/components/json-form/JS.tsx:2]: What is this file `JS` for ? @sagar0211

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,212 @@
+// TODO: See if the below LIB can be removed
```
Comment (saurshaz) [inline src/App.tsx:1]: @sagar0211 - things shall work fine without `App.tsx` being present

File: src/components/json-form/JS.tsx (hunk 1 above)
Comment (sagar0211) [reply src/components/json-form/JS.tsx]: Just took a reference. I deleted it.

File: src/App.tsx (hunk 2 above)
Comment (sagar0211) [reply src/App.tsx]: Okay . removed and pushed

**Pull request merged**